### DISCUSSION TITLE
What is everyone's favorite way to connect to PostgreSQL?

### DISCUSSION THREAD

**Discussion by HarrisonHemstreet** [actix/actix-web#2738] [Q&A]
I'm a newcomer to Actix! I really love the performance metrics and I thought I would give it a go!

I'm currently struggling to find a way to get Actix-Web to get data from my postgres database AND return the result via JSON though hitting a route.

What is the simplest way to do this? Are there any up to date examples on how? Please let me know! Thank you!

**Comment by asonix** [comment]
Lots of people like `sqlx` these days, but `diesel`, `seaorm`, and `tokio-postgres` are also good options

**Comment by borngraced** [comment]
tokio-postgres one way for me..
